components/Banner/examples/BannerTypicalUseExampleViewController.m

```diff
@@ -129,230 +130,238 @@ + (NSDictionary *)catalogMetadata {
   };
 }
 
+#pragma mark - UIViewController
+
+- (void)viewDidLayoutSubviews {
+  [super viewDidLayoutSubviews];
+
+  self.contentView.frame = self.view.bounds;
+  self.contentViewLabel.center = self.contentView.center;
+  CGRect exampleListTableViewFrame =
+      CGRectMake(0, self.view.bounds.size.height - exampleListTableViewHeight,
+                 self.view.bounds.size.width, exampleListTableViewHeight);
+  self.exampleListTableView.frame = exampleListTableViewFrame;
+
+  CGSize bannerViewSize =
+      [self.bannerView sizeThatFits:CGSizeMake(CGRectGetWidth(self.view.bounds), CGFLOAT_MAX)];
```

Comment:
Why CGFloat_MAX?  Wouldn't you want the banner to fit within the view, or does is it placed within a scroll view?

Reply:
Addressed in ccab2788839d0a28c0b4925a667ddf699304cdfe. This one is not placed in a scroll view.